### PR TITLE
LPD-41557 and LPD-43988 GET list and POST operations for Collection and Content Pages

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPD-41557
https://liferay.atlassian.net/browse/LPD-43988

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/headless/headless-admin-site/headless-admin-site-impl/src/main/java/com/liferay/headless/admin/site/internal/resource/v1_0/SitePageResourceImpl.java [hunk 1]
```diff
@@ -261,6 +224,157 @@ public Page<SitePage> read(
...
+
+		return UnicodePropertiesBuilder.create(
+			true
+		).setProperty(
+			LayoutTypePortletConstants.LAYOUT_TEMPLATE_ID,
+			widgetPageSettings.getLayoutTemplateId()
+		).buildString();
+	}
+
+	private boolean _inHiddenFromNavigation(PageSettings pageSettings) {
```
Comment (ruben-pulido) [inline modules/apps/headless/headless-admin-site/headless-admin-site-impl/src/main/java/com/liferay/headless/admin/site/internal/resource/v1_0/SitePageResourceImpl.java:370]: @lfbesada It looks like there is a typo. The intended method name is probably `isHiddenFromNavigation`

Comment (lfbesada) [reply modules/apps/headless/headless-admin-site/headless-admin-site-impl/src/main/java/com/liferay/headless/admin/site/internal/resource/v1_0/SitePageResourceImpl.java]: Thanks for catching it :heart: Fixed and rebased!

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
@ruben-pulido I Just added the last commit. My bad for missing that in the tests yesterday :disappointed:

**Comment by ruben-pulido** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**